docs/make.jl

```diff
@@ -123,6 +123,13 @@ end
 # link checking. Inn production we replace this by running the LaTeX build.
 write(joinpath(@__DIR__, "src", "MathOptInterface.pdf"), "")
 
+DocMeta.setdocmeta!(
+    MathOptInterface,
+    :DocTestSetup,
+    :(import MathOptInterface as MOI);
```

Comment:
I have to add it here because `_reinterpret_unsafe` is not added on any page.

Reply:
Maybe we can remove it from all the pages now by the way

Reply:
This should be a separate PR. Keep `import MathOptInterface as MOI` in the docstring for now.